package.json

```diff
@@ -11,9 +11,11 @@
 		"native-base": "^2.3.3",
 		"react": "16.0.0",
 		"react-native": "0.50.3",
+		"react-native-firebase": "^3.1.1",
```

Comment:
i think we dont use react-native-firebase so should be removed

Reply:
Done